### PR TITLE
Show milestones on main page

### PR DESCRIPTION
**Pull request by antooni**
This PR aims to create milestones for the whole layer 2 industry which will be shown on the main page. 

image

Resolves L2B-530

### CODE REVIEW

File: packages/config/src/layer2s/common/milestones.ts [hunk 1]
```diff
@@ -7,3 +9,62 @@ const MAINNET_OPEN = {
...
+    link: 'https://twitter.com/fuellabs_/status/1344707195250896899',
+    description:
+      'Fuel v1 is live, bringing payments scalability using Optimistic Rollup technology and UTXO model.',
+  },
+  {
+    name: 'Optimistic Rollup (universal)',
+    link: 'https://medium.com/ethereum-optimism/mainnet-soft-launch-7cacc0143cd5',
+    date: '2021-01-16T00:00:00Z',
+    description:
+      'Optimism sof launch is live, bringing first universal rollup based on optimistic architecture.',
```
Comment (krzkaczor) [inline packages/config/src/layer2s/common/milestones.ts:40]: sof -> soft typo

File: packages/config/src/layer2s/common/milestones.ts [hunk 2]
```diff
@@ -7,3 +9,62 @@ const MAINNET_OPEN = {
...
+  },
+  {
+    name: 'ZK Rollup (privacy)',
```
Comment (krzkaczor) [inline packages/config/src/layer2s/common/milestones.ts:43]: old: ZK Rollup (privacy)
new: ZK Rollup (privacy/payments) 
alternative version: ZK Rollup (anonymous payments)

**Review by krzkaczor — CHANGES_REQUESTED** [review]

File: packages/config/src/layer2s/common/milestones.ts [hunk 3]
```diff
@@ -7,3 +9,62 @@ const MAINNET_OPEN = {
 export const MILESTONES = {
   MAINNET_OPEN,
 }
+
+export const HOMEPAGE_MILESTONES: Milestone[] = [
+  {
+    name: 'ZK rollup (payments)',
```
Comment (luke7211) [inline packages/config/src/layer2s/common/milestones.ts:15]: First zkRollup (for payments)

File: packages/config/src/layer2s/common/milestones.ts [hunk 4]
```diff
@@ -7,3 +9,62 @@ const MAINNET_OPEN = {
...
+  },
+  {
+    name: 'Rollup centric vision for Ethereum',
+    date: '2020-10-02T00:00:00Z',
+    link: 'https://ethereum-magicians.org/t/a-rollup-centric-ethereum-roadmap/4698',
+    description:
+      'Rollups are considered a scaling solution for the near and mid-term future of Ethereum.',
+  },
+  {
+    name: 'Optimistic Rollup (payments)',
```
Comment (luke7211) [inline packages/config/src/layer2s/common/milestones.ts:29]: First fully-trustless Optimistic Rollup (for payments)

File: packages/config/src/layer2s/common/milestones.ts [hunk 5]
```diff
@@ -7,3 +9,62 @@ const MAINNET_OPEN = {
...
+    date: '2020-12-31T00:00:00Z',
+    link: 'https://twitter.com/fuellabs_/status/1344707195250896899',
+    description:
+      'Fuel v1 is live, bringing payments scalability using Optimistic Rollup technology and UTXO model.',
+  },
+  {
+    name: 'Optimistic Rollup (universal)',
```
Comment (luke7211) [inline packages/config/src/layer2s/common/milestones.ts:36]: First generalised Optimistic Rollup with fraud proofs

File: packages/config/src/layer2s/common/milestones.ts [hunk 6]
```diff
@@ -7,3 +9,62 @@ const MAINNET_OPEN = {
...
+  },
+  {
+    name: 'ZK Rollup (anonymous payments)',
+    link: 'https://medium.com/aztec-protocol/launching-aztec-2-0-rollup-ac7db8012f4b',
+    date: '2021-03-15T00:00:00Z',
+    description:
+      'Aztec is live, bringing private and scalable transactions using ZK Rollup technology.',
+  },
+  {
+    name: 'Public Optimistic Rollup (universal)',
```
Comment (luke7211) [inline packages/config/src/layer2s/common/milestones.ts:50]: First permissionless, generalised Optimistic Rollup open to anyone

File: packages/config/src/layer2s/common/milestones.ts [hunk 7]
```diff
@@ -7,3 +9,62 @@ const MAINNET_OPEN = {
...
+  },
+  {
+    name: 'Optimistic Rollup (universal)',
+    link: 'https://medium.com/ethereum-optimism/mainnet-soft-launch-7cacc0143cd5',
+    date: '2021-01-16T00:00:00Z',
+    description:
+      'Optimism soft launch is live, bringing first universal rollup based on optimistic architecture.',
+  },
+  {
+    name: 'ZK Rollup (anonymous payments)',
```
Comment (luke7211) [inline packages/config/src/layer2s/common/milestones.ts:43]: First privacy-focused Zk Rollup (payments)

**Review by luke7211 — APPROVED** [review]

**Pull request merged**